lib/qless/server.rb

```diff
@@ -17,6 +18,8 @@ class Server < Sinatra::Base
     # I'm not sure what this option is -- I'll look it up later
```

Comment:
You can probably remove this then? 😄

Reply:
That was in reference to `set :static, true`, I think.

Reply:
Yes, those two lines can be removed now?

Reply:
I guess if nobody looked it up in the last 6 years it's not likely to happen. :smile: